sdk/servicebus/ci.functions.yml

```diff
@@ -11,6 +11,7 @@ trigger:
     - sdk/servicebus/service.projects
     - sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus
     - sdk/servicebus/ci.functions.yml
+    - sdk/servicebus/test-resources.json
```

Comment:
Can we also include ServiceBusTestEnvironment path?

Reply:
I just put the whole test infrastructure folder